SCClassLibrary/Common/Control/Buffer.sc

```diff
@@ -67,7 +69,7 @@ Buffer {
 
 	allocMsg { arg completionMessage;
 		this.cache;
-		^["/b_alloc", bufnum, numFrames.asInteger, numChannels.asInteger, completionMessage.value(this)]
+		^["/b_alloc", bufnum, numFrames.asInteger, numChannels.asInteger, completionMessage.value(this), sampleRate]
```

Comment:
Here we also need to do `sampleRate !? _.asFloat ?? 0.0`!

Reply:
We also need to update the documentation for `allocMsg`!

Reply:
all done, except the 'doc' of 'allocMsg' which is already quite terse and has none of the arguments (so no need to add anything)

Reply:
Sorry, I get confused for a moment :) You are absolutely right.